### PR TITLE
fix(markdown-remark): mark prism package as an internal dependency

### PR DESCRIPTION
**Pull request by V3RON**
## Changes

This package wasn't marked as an internal dependency correctly, which introduced a race condition in the build process. Starting from now, `@astrojs/prism` will always be built first.

See discussion on Discord (contribute) for details.

## Testing

Clear Turborepo cache, try building `@astrojs/markdown-remark` - you will see an error regarding `@astrojs/prism`.

Checkout to this branch, do the same steps - you will see no error, `@astrojs/prism` will be built first.

## Docs

No changes needed

### CODE REVIEW

File: .changeset/few-beans-tickle.md
Comment (bluwy) [inline .changeset/few-beans-tickle.md:1]: I don't think we need a changeset here as the published version should still be the same for `@astrojs/prism`. So there's no change if we do another publish.

Comment (V3RON) [reply .changeset/few-beans-tickle.md:1]: I thought so, but wasn't sure if it's OK to have a PR without a changeset. Should I simply remove it?

Comment (bluwy) [reply .changeset/few-beans-tickle.md:1]: Yeah you can remove it. We only need a changeset if it requires publishing a release and it affects users.

Comment (V3RON) [reply .changeset/few-beans-tickle.md:1]: Done!

**Review by bluwy — APPROVED** [review]

**Pull request merged**